Game.java

```diff
@@ -62,7 +62,7 @@ public void playGame(Board board) {
         assignPlayers();
         print(gameMode.toUpperCase() + " game is starting..");
         print("Where would you like to go? (1, 2, 3, 4, 5, 6, 7, 8, 9)");
-        while (!board.checkForDraw()) {
+        while (!board.checkForDraw() || !board.getState().equals("Game Over")) {
```

Comment:
Ah here it is, from above where you construct a Board object with 'Game Over'...  I don't like this... are there any other states you construct it with?  It might be better to have a method like board.isGameOver(), which checks the board array to see if the game is over..